cmd/yurthub/app/config/config.go

```diff
@@ -94,6 +97,9 @@ func Complete(options *options.YurtHubOptions) (*YurtHubConfiguration, error) {
 		HubAgentDummyIfName:         options.HubAgentDummyIfName,
 		StorageWrapper:              storageWrapper,
 		SerializerManager:           serializerManager,
+		CAFile:                      options.CAFile,
+		CertFile:                    options.CertFile,
+		KeyFile:                     options.KeyFile,
 	}
```

Comment:
I think the establishment and configuration of TLS may be better done by yurthub, users do not need to pay attention to the underlying certificate configuration by default.

Reply:
It's a good idea. I will improve it later.